CppcheckTargets.cmake

```diff
@@ -136,7 +145,9 @@ function(add_cppcheck _name)
 			"add_cppcheck given a target name that does not exist: '${_name}' !")
 	endif()
 	if(CPPCHECK_FOUND)
-		set(_cppcheck_args)
```

Comment:
This looks like the patch above, which probably means this code could use some refactoring to reduce duplication.